tests/test_layering.py

```diff
@@ -37,62 +38,78 @@ def test_change_layer():
     for sprite in ones:
         scene.add(sprite)
 
-    assert list(scene)[0] == test_sprite
+    assert list(scene.sprite_layers())[0] == test_sprite
 
     test_sprite.layer = 2
 
-    assert list(scene)[-1] == test_sprite
+    assert list(scene.sprite_layers())[-1] == test_sprite
 
 
-def test_registered_layers():
+def test_defined_layers():
 
-    scene = scenes.BaseScene()
+    class LayeredScene(scenes.BaseScene):
+        defined_layers = {
```

Comment:
I hate this name

(I make this comment fully aware of the importance of names in discussions.)

Reply:
Um. . . `named_layers` maybe? It invites confusion with `layer_names` though.

Reply:
I figured just `layers`?